.github/workflows/code-style.yml

```
@@ -9,51 +9,37 @@ jobs:
 
     steps:
     - uses: actions/checkout@v1
-    - name: Set up Ruby
-      uses: actions/setup-ruby@v1
+
+    - name: Install PostgreSQL client
+      run: sudo apt-get install libpq-dev
+
+    - name: Cache Gems
+      uses: actions/cache@v1
       with:
-        ruby-version: 2.5.x
-    - name: Run code style checks
-      run: |
-        cd psd-web
-        gem install rubocop -v 0.75.1 --no-doc
-        gem install rubocop-performance -v 1.5.0 --no-doc
-        gem install rubocop-rails -v 2.3.2 --no-doc
-        gem install rubocop-rspec -v 1.36.0 --no-doc
-        gem install govuk-lint -v 4.0.1 --no-doc
-        rubocop
-  slim:
-    name: "Slim templates"
-    runs-on: ubuntu-latest
+        path: psd-web/vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gems-
```

should this be:

```suggestion
          ${{ runner.os }}-build-${{ env.cache-name }}-
```

...like in https://help.github.com/en/actions/automating-your-workflow-with-github-actions/caching-dependencies-to-speed-up-workflows ?